### PR TITLE
fix: remove lint warnings, update tests and rename file

### PR DESCRIPTION
**Pull request by zubhav**
**What kind of change does this PR introduce?**
Tidy-up/fixes

**Summary**
- Removes ESLint warnings that appear in CI
- Rename consants to constants in filename
- Remove unused file (TBC)

166890080-38cc2aa7-b70e-4897-ba87-ceab322b4fb2

### CODE REVIEW

File: packages/cli/babel/index.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// Is this file used/required? If not, can it be removed?
```
Comment (zubhav) [inline packages/cli/babel/index.js:1]: Probably a question for @rschristian 

I couldn't find any instance of this file being used. Unless it's used implicitly by Babel, could we remove it entirely?

Comment (rschristian) [reply packages/cli/babel/index.js:1]: Not used by us, no. However, users can add it as a preset to their own babel config if they so wish. 

It's not a great setup tbh, so I'm going to try to remove it in the next major, but there are some users who depend on it for now.

File: packages/cli/tests/build.test.js [hunk 2]
```diff
@@ -182,8 +182,10 @@ describe('preact build', () => {
 			`${dir}/build/${bundleFile}`,
 			'utf8'
 		);
+
 		// "Hello World!" should replace 'process.env.PREACT_APP_MY_VARIABLE'
-		expect(transpiledChunk.includes('console.log("Hello World!")')).toBe(true);
+		expect(transpiledChunk.includes('Hello World!')).toBe(true);
```
Comment (rschristian) [inline packages/cli/tests/build.test.js:187]: nit: We do need to check for quotes when replacing env vars, else if we fail to stringify we'd end up with `var foo = Hello World!`

```suggestion
		expect(transpiledChunk.includes('"Hello World!"')).toBe(true);
```

File: packages/cli/babel/index.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
...
+
```
Comment (rschristian) [inline packages/cli/babel/index.js:2]:
```suggestion
```

**Review by rschristian — APPROVED** [review]
Thanks for doing this!

**Pull request merged**